ush/compare_f90nml.py

```diff
@@ -78,7 +78,7 @@ def _print_diffs(diff_dict: Dict) -> None:
             for kk in diff_dict[path].keys():
                 items = diff_dict[path][kk]
                 print(
-                    f"{kk:>{max_len+2}} : {' | '.join(map(str, diff_dict[path][kk]))}")
+                    f"{kk:>{max_len + 2}} : {' | '.join(map(str, diff_dict[path][kk]))}")
```

Comment:
@WalterKolczynski-NOAA 
Any idea why the pynorms test did not catch this before?

Reply:
Nope. Maybe it ignores it in f-strings (or otherwise has trouble parsing them)?